ingestion_server/ingestion_server/indices/mappings/common.json

```diff
@@ -0,0 +1,108 @@
+{
+  "authority_boost": {
+    "type": "rank_feature"
+  },
+  "authority_penalty": {
+    "positive_score_impact": false,
+    "type": "rank_feature"
+  },
+  "category": {
+    "type": "keyword"
+  },
+  "created_on": {
+    "type": "date"
+  },
+  "creator": {
+    "fields": {
+      "keyword": {
+        "ignore_above": 256,
+        "type": "keyword"
+      }
+    },
+    "type": "text"
+  },
+  "description": {
+    "analyzer": "custom_english",
+    "fields": {
+      "keyword": {
+        "ignore_above": 256,
+        "type": "keyword"
+      }
+    },
+    "similarity": "boolean",
+    "type": "text"
+  },
+  "extension": {
```

Comment:
When does it make sense to have full-text search for extensions? I would think that keyword search for extension should be enough...

Reply:
This was for some very nice cases like `jpg/jpeg`. I might be overthinking this though.

Reply:
Since we are processing the file name to get the extension, we can also do replacing there. I don't think that for the user it matters if they find `jpg` and not `jpeg` as they asked, for instance. Does it??

Reply:
I'm not sure about the existing index, though. We could add both `jpg` and `jpeg` to the filter as an interim solution, before re-indexing, if necessary.

Reply:
That was a very uncommon example, we might not even have to remediate this. Changed to `keyword` in 71e74d9.